knowledge-base/why-get-document-request-is-unauthorized.md

```diff
@@ -29,7 +29,7 @@ When performing security tests with automated tools, there may be indications of
 The [Get Document](../telerik-reporting-rest-documents-api-get-document) request (e.g. "/api/reports/clients/031206-6569/instances/031209-532c/documents/031210-54b5031210-5509") is performed to download the generated report document as an attachment (e.g. PDF file). Importantly, the __JavaScript doesn't have access to the client machine file system__, so it is necessary to rely on the browser to actually download the file, e.g. to open the File Save dialog. Therefore, the Html5 Report Viewer downloads the rendered reports through the [window.open() method](https://developer.mozilla.org/en-US/docs/Web/API/Window/open). However, there are no options to add headers as this is not an AJAX request, hence if the endpoint was secured, it was not going to be possible to open/download the generated report through the viewer.
```

Comment:
slug for [Get Document]
telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/documents-api/get-document